frontend/packages/gitops-plugin/src/components/utils/gitops-utils.ts

```diff
@@ -62,11 +62,15 @@ export const fetchAllAppGroups = async (baseURL: string, manifestURLs: string[],
   return [allAppGroups, emptyMsg];
 };
 
-export const getEnvData = async (envURI: string, env: string, appURI: string) => {
+export const getEnvData = async (v2EnvURI: string, envURI: string, env: string, appURI: string) => {
   let data;
   try {
-    data = await coFetchJSON(`${envURI}/${env}${appURI}`);
-  } catch {} // eslint-disable-line no-empty
+    data = await coFetchJSON(`${v2EnvURI}/${env}${appURI}`);
+  } catch {
+    try {
+      data = await coFetchJSON(`${envURI}/${env}${appURI}`);
+    } catch {} // eslint-disable-line no-empty
```

Comment:
Another unhandled error block.

Reply:
Added to issue [GITOPS-1278](https://issues.redhat.com/browse/GITOPS-1278)